### PR TITLE
added new file main.js

### PR DESCRIPTION
**Pull request by Irina-pi**

### CODE REVIEW

**Review by 5tronciy — CHANGES_REQUESTED** [review]
.gitignore is missed.
Also there is no package.json in this branch.
Add these and other necessary files.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+  <title>IF-JS</title>
+</head>
+<body>
+</body>
+</html>
```
Comment (5tronciy) [inline index.html]: Looks like you don't know how to use prettier.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Husky installing is required.
And use the packages you installed while you can't do without them.

File: index.html (hunk 1 above)
Comment (Irina-pi) [reply index.html]: sorry, I forget corrected.

Comment (Irina-pi) [reply index.html]: It is ready.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Looks good!
After approval and only after it, you may merge PR yourself. Let me know if something goes wrong with it.

**Review by 5tronciy — APPROVED** [review]

**Pull request merged**